### DISCUSSION TITLE
Interested in non-curl-bash installation methods?

### DISCUSSION THREAD

**Discussion by shanesveller** [fluxcd/flux2#176] [Q&A] [answered]
I would love to be able to onboard my teammates with i.e. a Homebrew tap, and for now I document the (appreciably simple) steps of downloading and moving/chmod'ing the binary. I'd possibly be interested in contributing this here if the idea itself is welcome, but I'll acknowledge it comes with an upkeep cost that may not be justified at this stage.

As an example of prior art, Tekton maintains their Homebrew tap [here](https://github.com/tektoncd/homebrew-tools/blob/master/Formula/tektoncd-cli.rb).

Usage looks like:
```shell
brew tap tektoncd/tools
brew install tektoncd-cli
```

And then you're able to get update notifications and perform upgrades as usual.

**Comment by hiddeco** [comment, marked as answer]
Given [there is a brew recipe for `fluxctl`](https://github.com/Homebrew/homebrew-core/blob/b5d1ecb4880401bb4e2e8b9144ac87207d668467/Formula/fluxctl.rb) in `homebrew-core`, and [a Chocolatey package for Windows](https://chocolatey.org/packages/fluxctl) (currently blocked by #47), I think this would be a welcome addition.

**Reply by stefanprodan** [reply]
We now have homebrew support for Linux amd64/arm64 and macOS:

```sh
brew tap fluxcd/tap
brew install gotk
```
